src/ui/components/URLBar/URLBar.tsx

```diff
@@ -104,7 +104,7 @@ export function URLBar() {
             <>
               <BackButton
                 onClick={() => navigate(getBackOrHome() as number)}
-                title={`Press "backspace" to navigate back`}
+                title={'Press "backspace" to navigate back'}
```

Comment:
We can remove braces here

Reply:
But there are double quotes in the message

Reply:
Oh, agree